core-bundle/src/Resources/contao/dca/tl_user.php

```diff
@@ -609,6 +609,11 @@ public function addTemplateWarning()
 		{
 			Message::addInfo($GLOBALS['TL_LANG']['MSC']['userThemeImport']);
 		}
+
+		if ($objResult->showUnfilteredHtmlWarning > 0)
```

Comment:
```suggestion
		if ($objResult->showUnfilteredHtmlWarning)
```

We are not using `> 0` above, either. Not sure which variant is correct, though.